### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by SaraDPH**
Updated to be more evergreen for those looking for Pulumi + Java + IaC, removing time references like "today" and "announcing." Some rearrangement to make sense, bottom link removal of not clicked link and may 5th workshop

### CODE REVIEW

**Comment by susanev** [comment]
id perfer to wait on @mikhailshilkov's approval on this one, also it looks like you have a build failure

**Comment by SaraDPH** [comment]
> id perfer to wait on @mikhailshilkov's approval on this one, also it looks like you have a build failure

What does "build failure" mean? @mikhailshilkov can you please review, change and fix this? Thanks!
The article is being linked in a lead form and shared on social.

File: themes/default/content/blog/announcing-infrastructure-as-code-with-java-and-pulumi/index.md
Comment (SaraDPH) [inline themes/default/content/blog/announcing-infrastructure-as-code-with-java-and-pulumi/index.md:125]:
```suggestion
Pulumi is [open source on GitHub](https://github.com/pulumi/pulumi) and you can find the Java plugin at [pulumi/pulumi-java](https://github.com/pulumi/pulumi-java).
And you're welcome to [join the community in Slack](https://slack.pulumi.com/) to discuss your scenarios, ideas, and to get any needed assistance from the team and other end users.
```

**Comment by susanev** [comment]
Sara and i zoomed and fixed the build failure together

**Review by justinvp — APPROVED** [review]
LGTM

**Pull request merged**
